packages/devel/autoconf/package.mk

```
@@ -1,12 +1,13 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="autoconf"
-PKG_VERSION="2.71"
-PKG_SHA256="f14c83cfebcc9427f2c3cea7258bd90df972d92eb26752da4ddad81c87a0faa4"
+PKG_VERSION="2.72"
+PKG_SHA256="ba885c1319578d6c94d46e9b0dceb4014caafe2490e437a0dbca3f270a223f5a"
 PKG_LICENSE="GPL"
-PKG_SITE="http://sources.redhat.com/autoconf/"
-PKG_URL="http://ftpmirror.gnu.org/autoconf/${PKG_NAME}-${PKG_VERSION}.tar.xz"
+PKG_SITE="https://sources.redhat.com/autoconf/"
```

```suggestion
PKG_SITE="https://www.gnu.org/software/autoconf/"
```
the redhat site is not working, gnu site works and has all the infos + links